front/src/views/user/eating/eating.view.html

```diff
@@ -0,0 +1,109 @@
+<div>
+  <mmt-stepper></mmt-stepper>
+  <v-row>
+    <v-col md="2" sm="12" class="d-flex justify-center align-center">
+      {{$t('eating.results')}} ({{scheduledPlacesList.length}})
+    </v-col>
+    <v-col md="6" sm="12" />
+    <v-col md="2" sm="12" class="d-flex justify-center align-center">
+      <v-btn rounded small color="accent lighten-1" class="mmt-btn" width="165" height="39"
+        @click="$router.push({name: 'ScheduleEatingPlaceNew'})">
+        {{$t('eating.add')}}
+      </v-btn>
+    </v-col>
+    <v-col md="2" sm="12" class="d-flex justify-center align-center">
+      <v-btn rounded small color="accent lighten-1" class="mmt-btn" width="110" height="39">
+        {{$t('eating.filter')}}
+        <v-icon dense right>mdi-tune</v-icon>
+      </v-btn>
+    </v-col>
+  </v-row>
+  <v-row>
+    <v-col class="d-flex flex-wrap">
+      <v-card :key="index" v-for="(place, index) in scheduledPlacesList"
+        :flat="place.serverInfo.id!==selectedId"
+        class="place-card" height="300" width="259" @click="onSelectPlace(place)"
+        :class="{ 'active-card': place.serverInfo.id === selectedId }">
+        <v-img v-if="place.googleDetails && place.googleDetails.photos" 
+          :aspect-ratio="16/9"
+          :lazy-src="defaultImage" :src="place.googleDetails.photos[0].getUrl(259)"
```

Comment:
getUrl - fullPath filter (it seems we have one)

Reply:
259 - size image

Reply:
const - and name it well, please